crates/sparrow-runtime/src/prepare/prepare_parquet.rs

```diff
@@ -0,0 +1,19 @@
+use error_stack::ResultExt;
+use sparrow_api::kaskada::v1alpha::{slice_plan, PreparedFile, TableConfig};
+
+use crate::prepare::Error;
+use crate::read::ParquetFile;
+use crate::stores::{ObjectStoreRegistry, ObjectStoreUrl};
+
+pub async fn prepare_parquet<'a>(
```

Comment:
Is this used elsewhere related to this change?

Reply:
Nope. This file snuck in from an unrelated branch.